cve_bin_tool/cvedb.py

```diff
@@ -47,42 +32,37 @@
 )
 DBNAME = "cve.db"
 OLD_CACHE_DIR = os.path.join(os.path.expanduser("~"), ".cache", "cvedb")
-NVD_FILENAME_TEMPLATE = "nvdcve-1.1-{}.json.gz"
 
 
 class CVEDB:
     """
-    Downloads NVD data in json form and stores it on disk in a cache.
+    Retrieves CVE data from sources and handles CVE Database
```

Comment:
Maybe say explicitly say where to find the data sources in the source tree here.  
```suggestion
    Retrieves CVE data from data sources and handles CVE Database.
    The sources can be found in the cve_bin_tool/data_sources/ directory.
```